### DISCUSSION TITLE
crouton rescue kit: commands to make installing and using crouton easier

### DISCUSSION THREAD

**Discussion by CroutonIsFun** [dnschneid/crouton#5070] [Ideas]
Note: in process, not finished yet...

alias InstallCrouton="sudo install -Dt /usr/local/bin -m 755 ~/Downloads/crouton"
alias crouton="sudo CROUTON_BRANCH=silence /usr/local/bin/crouton"
alias ChromebrewInstall="bash <(curl -L git.io/vddgY) && . /.bashrc"
alias LegacyBoot="cd; curl -LO mrchromebox.tech/firmware-util.sh && sudo bash firmware-util.sh"

**Comment by CroutonIsFun** [comment]
Will open later, if appropriate.

**Comment by dnschneid** [comment]
Consider using the wiki?
